packages/vite/src/node/optimizer/resolve.ts

```diff
@@ -79,6 +79,7 @@ export function expandGlobIds(id: string, config: ResolvedConfig): string[] {
           const exportsValue = getFirstExportStringValue(exports[key])
           if (!exportsValue) continue
 
+          // TODO: Raise during the PR review.
           // "./dist/glob/*-browser/*.js" => "./dist/glob/**/*-browser/**/*.js"
           // NOTE: in some cases, this could expand to consecutive /**/*/**/* etc
           // but it's fine since fast-glob handles it the same.
```

Comment:
I've done some checking manually to see whether the behavior in `tinyglobby` matches `fast-glob's` in this case, but a concrete use case would be great to confirm. I would update this comment afterwards.

Reply:
This comment describes about [the behavior of `*` in `exports` field](https://nodejs.org/api/packages.html#subpath-patterns). `*` in exports matches any string including `/`, so `*` is replaced with `**/*` to match nested directories.

For example, for an input `./src/features/*.js` (gets replaced to `./src/features/**/*.js`), it should match

- `./src/features/x.js`
- `./src/features/y/y.js`

Probably the note is explaining about cases like:

- `./*/*.js` -> `./**/*/**/*.js`
- `./**.js` -> `./**/***/*.js`

I don't think anyone would use this as multiple `*` is only allowed for the righthand side (https://github.com/lukeed/resolve.exports/issues/9). But just need to be sure that we don't crash or error in that case.

cc @bluwy

Reply:
It seems like `tinyglobby` indeed works as expected in these cases:

```shell
> globSync("./assets/*/*.js")
[ 'assets/foo/foo.js' ]
> globSync("./assets/**/*/**/*.js")
[ 'assets/foo/foo.js' ]
> globSync("./assets/**.js")
[]
> globSync("./assets/**/***/*.js")
[ 'assets/foo/foo.js' ]
```

Reply:
Went ahead and removed the TODO comment now, please let me know if you'd like me to do some more testing here.